flow/templates/stampede2.sh

```diff
@@ -1,14 +1,30 @@
 {# Templated in accordance with: https://portal.tacc.utexas.edu/user-guides/stampede2 #}
 {% extends "slurm.sh" %}
+{% set ns = namespace(use_launcher=True) %}
+{% if operations|length() < 2 %}
+{% set ns.use_launcher = False %}
+{% endif %}
```

Comment:
I believe this should be an `{% else %}`, right?

Reply:
I think both logic gets the same result, now I am just checking each criteria one by one, it will do extra check even when operation.length=1 but given that I think it won't have significant slow down

Reply:
Ok, we can leave as is if you like.